src/components/Scrolltotop-button/ScrollToTopStyles.js

```diff
@@ -0,0 +1,38 @@
+import styled, { keyframes } from "styled-components";
+
+export const BackToTopButton = styled.button`
+    width: 3.5rem;
+    height: 3.5rem;
+    position : fixed;
+    bottom: 1rem;
+    right : 1.5rem;
+    outline : none;
+    border-radius:  50%;
+    border : none;
+    animation-name : ${props => props.isMounted ? onMountBtnAnimation : onUnmountBtnAnimation};
+    animation-duration: ${props => `${props.delay}ms`};
+    background-color: #3c494f;
+    box-shadow: 0 2px 5px 0 rgba(0,0,0,.26);
+    cursor: pointer;
+`;
```

Comment:
You need to increase the z-index for the button, it hides under other some sections across various pages.

Reply:
@Nikhil-Ladha done.